src/khoj/routers/api.py

```diff
@@ -603,7 +603,13 @@ async def chat(
         request, meta_log, q, (n or 5), (d or math.inf), conversation_command
     )
 
-    if conversation_command == ConversationCommand.Default and is_none_or_empty(compiled_references):
+    if conversation_command == ConversationCommand.Clear:
+        await ConversationAdapters.adelete_conversation_by_user(user)
+        return StreamingResponse(
+            iter(["Cleared conversation history"]), media_type="text/event-stream", status_code=200
+        )
```

Comment:
Shouldn't the UI update when this occurs? We probably want to handle it in a special way on the client side.

Reply:
This will be easier if we create a separate UI button for this action; it would clear the specific chat in the UI and send a delete request on the server side.

Reply:
Updated code to clear the conversation history displayed on the clients after the clear history action has successfully executed